tests/cookiesParser.js

```diff
@@ -0,0 +1,11 @@
+function getCookiesObj(cookiesArray) {
```

Comment:
@IgorChaikin Also could you move this file to `lib` or `utils` directory and cover it with tests, for example:

- Check that `cookiesArray` is `undefined`
- Check that `cookiesArray` is `null`
- Check that `cookiesArray` is empty array `[]`
- Correct case